.coveragerc

```diff
@@ -6,6 +6,7 @@ omit =
     server/config.py
     scripts/*
     fixtures/*
+    server/api/batch_inventory.py
```

Comment:
Punted on full test updating in this PR since I'm on a time crunch. But have tested manually and will do more manual testing before merging. And am planning to add automated tests, ideally before the actual audit next week